tests/map.spec.js

```diff
@@ -87,29 +88,29 @@ test('test traversableCellsInTile', t => {
     [1, 0, 0, 0, 0, 1],
     [1, 1, 0, 0, 1, 1],
   ];
-  t.same(helpers.traversableCellsInTile(false, 6, 14), expected);
+  t.same(map.traversableCellsInTile(false, 6, 14), expected);
 });
 
 
 test('test getTraversableCells', t => {
   t.plan(3);
 
   // create a dummy map from bombs, spikes, gates, and regular tiles
-  const bomb = helpers.tileTypes.BOMB;
-  const spike = helpers.tileTypes.SPIKE;
-  const redgate = helpers.tileTypes.RED_GATE;
-  const bluegate = helpers.tileTypes.BLUE_GATE;
-  const blank = helpers.tileTypes.REGULAR_FLOOR;
+  const bomb = tileTypes.BOMB;
+  const spike = tileTypes.SPIKE;
+  const redgate = tileTypes.RED_GATE;
+  const bluegate = tileTypes.BLUE_GATE;
+  const blank = tileTypes.REGULAR_FLOOR;
 
   // initialize current player as red
-  const me = { team: helpers.RED_TEAM };
+  mockMe();
```

Comment:
it seems like mockMe should take in the color of the player as an input. Can you add a few tests for if the current player is blue? (can use the same map we already defined here, just the expected output for `getTraversableTiles` would be different)